<launch>

  <!-- By default, we are not in debug mode -->
  <arg name="mode" default="1" />
  <arg name="debug" default="false" />
  <arg name="verbose" default="1" />
  <arg name="use_experience" default="1" />
  <arg name="order" default="$(find picknik_main)/orders/simple.json" />
  <arg name="order_start" default="0" />
  <arg name="jump_to" default="0" />
  <arg name="show_database" default="0" />

  <!-- Ignore for now -->
  <arg name="saving_enabled" default="1" />
  <arg name="use_scratch" default="1" />

  <!-- Load the URDF, SRDF and other .yaml configuration files on the param server -->
  <include file="$(find jaco_moveit_config)/launch/planning_context.launch">
    <arg name="load_robot_description" value="true"/>
  </include>


  <!-- Run the main MoveIt executable without trajectory execution (we do not have controllers configured by default) -->
  <include file="$(find picknik_main)/launch/jaco_apc.launch">
    <arg name="mode" value="$(arg mode)"/>
    <arg name="debug" value="$(arg debug)"/>
    <arg name="verbose" value="$(arg verbose)"/>
    <arg name="use_experience" value="$(arg use_experience)"/>
    <arg name="order" value="$(arg order)"/>
    <arg name="order_start" value="$(arg order_start)"/>
    <arg name="jump_to" value="$(arg jump_to)"/>
    <arg name="show_database" value="$(arg show_database)"/>
    <arg name="fake_execution" value="true"/>
  </include>

</launch>
